crypto-core/src/main/java/com/palantir/crypto2/cipher/CipherStreamSupplier.java

```diff
@@ -1,17 +1,5 @@
 /*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
do you intentionally want to remove Apache license?

Reply:
fixed. thanks